### PR TITLE
Disable moose option for qnap build

### PR DESCRIPTION
**Pull request by matislovas**
### Problem
To fully integrate `moose` to `qnap` (and other embedded platforms), we have to integrate moose-worker, as well have a static build of `libsqlite3`.

### Solution
Moose integration was postponed, so building qnap without `--moose` option.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Comment by matislovas** [comment]
TODO: adjust `libtelio-build` tag before merge.

**Review by tomaszklak — COMMENTED** [review]
+1

File: ci/moose_utils.py [hunk 1]
```diff
@@ -42,10 +42,7 @@ def _download_moose_file(opsys: str, arch: str, file_name: str):
     if nexus_url is None:
         raise ValueError("LIBTELIO_ENV_SEC_NEXUS_URL not set")
 
-    if opsys == "qnap":
-        url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/linux/{arch}/{file_name}"
-    else:
-        url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/{opsys}/{arch}/{file_name}"
+    url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/{opsys}/{arch}/{file_name}"
```
Comment (LukasPukenis) [inline ci/moose_utils.py:45]: `opsys` stands for OS I suppose, maybe worth renaming?

**Review by LukasPukenis — COMMENTED** [review]
At a glance it looks correct +1

File: ci/moose_utils.py (hunk 1 above)
Comment (matislovas) [reply ci/moose_utils.py]: Yeah, a little confusing, renamed

**Pull request merged**